SwiftProtobuf.podspec

```diff
@@ -8,16 +8,14 @@ Pod::Spec.new do |s|
   s.source = { :git => 'https://github.com/gojekfarm/swift-protobuf.git', :tag => s.version }
 
   s.requires_arc = true
-  s.ios.deployment_target = '11.0'
-  s.osx.deployment_target = '10.13'
-  s.tvos.deployment_target = '11.0'
-  s.watchos.deployment_target = '4.0'
-  s.visionos.deployment_target = "1.0"
+  s.ios.deployment_target = '9.0'
```

Comment:
Let's revert this particular commit, these changes don't seem necessary.

From the diff, it looks like we are downgrading cocoapods_version, and `deployment_target`